src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmLevitsky.cpp

```diff
@@ -133,7 +133,10 @@ char MSAConsensusAlgorithmLevitsky::getConsensusChar(const MultipleAlignment& ma
 
     // count local freqs first
     QVarLengthArray<int> localFreqs(256);
-    memset(localFreqs.data(), 0, localFreqs.size() * 4);
+    for (int i = 0; i < 256; i++) {
```

Comment:
Why did you replace the memset with this boilerplate code? If you do not link old-style memset you can use std::fill 1-liner

Reply:
I can't get what you mean. `QVarLengthArray` don't have method like 'fill', and we cant call std::fill for qt container.
About previous code, i don't like way to operate with container content throu the memory modification even it possible with container methods. At least if you want keep previous code it should be `memset(localFreqs.data(), 0, localFreqs.size() * sizeof(int));`.

Reply:
https://en.cppreference.com/w/cpp/algorithm/fill